test/TestServerProjects/body-datetime/body_datetime.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.13.0" />
+    <PackageReference Include="Azure.Core.Experimental" Version="0.1.0-preview.12" />
```

Comment:
We shouldn't add an LLC reference to projects that dont' need it.

Reply:
Agreed. Let me look into this.

Reply:
Fixed.